templates/tutorialv2/stats/index.html

```diff
@@ -90,14 +90,13 @@ <h2 class="ico-after ico-tutorials">
             {% if display == 'global' %}
                 <button type="submit" class="btn">{% trans "Comparer" %}</button>
                 </form>
+            {% elif not content.has_extracts %}
```

Comment:
oui mais non. car sur un big tuto, quand tu es sur la page globale tu ne dois pas avoir ce bouton.

Reply:
Je suis d'accord et normalement mon code prend en compte ça :) `{% elif not content.has_extracts %}` c'est comme si j'avais mis `{% if display != 'global' and not content.has_extracts %}` !